server/src/main/kotlin/fi/oph/kitu/kielitesti/MoodleService.kt

```diff
@@ -26,7 +30,7 @@ data class MoodleUserList(
         fun tryParse(json: String): List<Oppija> {
```

Comment:
Suoritusten parsimisen voisi tehdä kans näin, niin Moodlesta tulevat mahdolliset virheviestit saadaan kiinni ja logitettua (Moodle palauttaa aina http statuksen 200, vaikka vastauksena olisi virhe)

Reply:
Onks tää nyt pularin uusimmassa versiossa ok?